packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -198,6 +206,9 @@ contract Bridge is EssentialContract, IBridge {
     }
 
     /// @inheritdoc IBridge
+    /// @dev This transaction's gas limit must not be smaller than:
+    /// `(message.gasLimit - GAS_RESERVE) * 24 / 23 + GAS_RESERVE`,
```

Comment:
```suggestion
    /// `(message.gasLimit - GAS_RESERVE) * 64 / 63 + GAS_RESERVE`,
```
I think these are the numbers, otherwise not sure how this formula was created (some doubling/approximation somewhere maybe? because the message data limit is also not taken into account here?).